Mapbox/MapboxMapsFoundation/Camera/CameraManager.swift

```diff
@@ -210,6 +210,7 @@ public class CameraManager {
                           pitch: CGFloat? = nil,
                           animated: Bool = false,
                           duration: TimeInterval? = nil,
+                          takeShortestPath: Bool = true,
```

Comment:
We need a better name for this -- `takeShortestPath` is too generic.

Reply:
Actually could you add this opt-out support as part of a separate PR? I think this needs some thought / standardization.

Reply:
I added this per @julianrex comment above. I agree that right now we shouldn't add that in given we have 2 setCamera methods and one of them will allow an opt out of optimization.